cartography/intel/gcp/compute.py

```diff
@@ -112,6 +136,62 @@ def transform_gcp_instance(instance, project_id, zone_name):
     return instance
 
 
+def transform_gcp_vpcs(vpc_res):
+    """
+    Transform the VPC response object for Neo4j ingestion
+    :param vpc_res: The return data
+    :return: List of VPCs ready for ingestion to Neo4j
+    """
+    vpc_list = []
+    prefix = vpc_res['id']
+    projectid = prefix.split('/')[1]
```

Comment:
adding examples would help understand/maintain the code

Reply:
I'll add a comment on the form of the prefix and what we're parsing out.